nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -159,3 +159,7 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - QtMultimedia has changed its default backend to `QT_MEDIA_BACKEND=ffmpeg` (previously `gstreamer` on Linux or `darwin` on MacOS).
   The previous native backends remain available but are now minimally maintained. Refer to [upstream documentation](https://doc.qt.io/qt-6/qtmultimedia-index.html#ffmpeg-as-the-default-backend) for further details about each platform.
+
+- The `drbd` package has been updated from `9.19.1` to `9.27.0`.
```

Comment:
The NixOS release notes don't need every version bump. The line below is appropriate however.